lstchain/image/muon/muon_analysis.py

```diff
@@ -40,30 +42,18 @@ def get_muon_center(geom, equivalent_focal_length):
     x, y:    `floats` coordinates in  the NominalFrame
     """
 
-    x, y = geom.pix_x, geom.pix_y
-
-    telescope_pointing = SkyCoord(
-            alt = 70 * u.deg,
-            az = 0 * u.deg,
-            frame = AltAz()
-        )
-
     camera_coord = SkyCoord(
-            x = x, y = y,
-            frame = CameraFrame(
-                focal_length = equivalent_focal_length,
-                rotation = geom.pix_rotation,
-                telescope_pointing = telescope_pointing
-            )
-    )
-    nom_coord = camera_coord.transform_to(
-            NominalFrame(origin=telescope_pointing)
+        x=geom.pix_x,
+        y=geom.pix_y,
+        frame=CameraFrame(
+            focal_length=equivalent_focal_length,
+            rotation=geom.cam_rotation,
         )
+    )
+    tel_coord = camera_coord.transform_to(TelescopeFrame())
 
-    x = nom_coord.delta_az.to(u.deg)
-    y = nom_coord.delta_alt.to(u.deg)
+    return tel_coord.delta_az, tel_coord.delta_alt
```

Comment:
This is why it's called delta. I came up with this due to lack of a better name. I didn't want to call it `x` and `y`, as this is for camera coordinates.

`delta_az` is aligned with the azimuth axis and `delta-alt` is aligned with the `altitude` axis.

So it's delta in an angular equivalence of `delta = position - pointing_position`

Reply:
Another point is that the results have changed.  There is a change of coordinates, ring centers are now in different locations (I did not check the exact transformation), though radii seem to be the same. More worringly, the selected rings (event_id's) changed. Could not check systematically, but: in many cases the non-matching events seem to be rings cut by the FoV edge which have very different reconstructed impact parameter (not strange, if ring is cut the i.p. cannot be well determined). Some others seem to be contained rings... And from other parameters there is no way to tell one version is better than the other.
Any idea of what may have changed? Did you expect the change of coordinates?

Reply:
So before, there was `rotation=geom.pix_rotation` which is for sure wrong. The rotation in the `CameraFrame` has nothing to do with pixel orientation.

Reply:
If any of `cam_rotation` or `pix_rotation` is non-zero, that could explain the differences.

Reply:
@moralejo When did you test? I made a mistake which I corrected in a second commit. Also, what data are you using for your tests, so I coudl crosscheck?

Reply:
> This is why it's called delta. I came up with this due to lack of a better name. I didn't want to call it `x` and `y`, as this is for camera coordinates.
> 

If the sign is the same as that of x and y (of some camera system) then I think it would be preferable, since those are about the correct axes.  The delta_az between two pixels should instead be the difference in their azimuth (of the usual horizontal coordinates). But anyway, fine for me just to add a comment on it.

Reply:
> @moralejo When did you test? I made a mistake which I corrected in a second commit. Also, what data are you using for your tests, so I coudl crosscheck?

I am using run 1881.0000 (with the calibration files produced by the onsite) But you need also the recent fixes in ctapipe's MuonLineIntegrate

This is what I run:
`python lstchain_data_r0_to_dl1.py --infile LST-1.1.Run01881.0000.fits.fz --outdir ./dl1_extractor_12_4/ --pedestal_path ../drs4_pedestal.Run1872.0000.fits --calibration_path ../calibration.Run1873.0000.hdf5 --time_calibration_path ../time_calibration.Run1625.0000.hdf5 --pointing_file_path ../drive_log_20_01_27.txt --config_file config_extractor_12_4.json`

The config file is the standard one, just changing window_width from 8 to 12.

Reply:
> @moralejo When did you test? I made a mistake which I corrected in a second commit. Also, what data are you using for your tests, so I coudl crosscheck?

I tested after your third commit. I saw a change related to camera rotation in the 4th commit. Shall I try?

Reply:
This was only in the plotting code, which did not ran before, so I assume you did not plot events.

Reply:
> The delta_az between two pixels should instead be the difference in their azimuth (of the usual horizontal coordinates). But anyway, fine for me just to add a comment on it.

I tend to agree here. I'll open an issue in ctapipe.

Reply:
> This was only in the plotting code, which did not ran before, so I assume you did not plot events.

Yes, just saw it. So any idea why coordinates changed? I thought it was just a code simplification.

Reply:
No, one of the first commits also changed it for the actual reconstruction code, the fourth commit changed it in the plotting code.

The important change is this:

`CameraFrame(rotation=geom.pix_rotation)` → `CameraFrame(rotation=geom.cam_rotation)`

The first is clearly wrong (and copy/paste from the same mistake in the ctapipe muon code

Reply:
See issue https://github.com/cta-observatory/ctapipe/issues/898

Reply:
> No, one of the first commits also changed it for the actual reconstruction code, the fourth commit changed it in the plotting code.
> 
> The important change is this:
> 
> `CameraFrame(rotation=geom.pix_rotation)` → `CameraFrame(rotation=geom.cam_rotation)`
> 
> The first is clearly wrong (and copy/paste from the same mistake in the ctapipe muon code

ok, my test was done with everything but the plotting changes, so it stands that there is a difference in the results (for good or for bad I cannot tell, they're very similar for most events...)